modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -19,6 +19,7 @@ For information about supported platforms and architecture, see the link:https:/
 .Procedure
 
 To enable node indexing, also known as node scanning, by using Scanner V4:
+//is this page/procedure still required since Scanner V4 is installed by default now on the secured cluster?
```

Comment:
I believe so. I think @vikin91 may be able to explain this more, though

Reply:
This procedure only applies to customers migrating from the Stackrox Scanner to Scanner v4, so mainly for versions 4.6 and 4.7 where that was not enabled by default. I think, we need to add a text explaining that or isolate the sections for enabling the node scanning, so that it does not confuses the readers.

In 4.8 Node Scanning with Scanner v4 is enabled and should work out of the box.

Reply:
Since these docs are only for 4.8 and later, I think this should be deleted then?

Reply:
Added a note - we can revise or delete later if needed.